### PR TITLE
Finish step submit

### PR DESCRIPTION
**Pull request by marberrym**
Finished step submit and project publishing, also edited the navbar to be more responsive.

Proud of the work we got done today.  Let's bring it all together tomorrow.

### CODE REVIEW

File: src/components/ThreeBarNav.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        this.state = {
+            isOpened: false
+        }
+    }
+    handleClickOutside() {
+        this.setState({isOpened: false});
+    }
+     
+    toggle() {
+        this.setState({isOpened: !this.state.isOpened});
```
[the remark this answers is not in the document]
Comment (marberrym) [reply src/components/ThreeBarNav.js]: fixed

File: src/pagescreens/EditProjectScreen.js [hunk 2]
```diff
@@ -89,6 +89,16 @@ class EditProjectScreen extends Component {
...
         }
+
+        let publishProject = () => {
+            fetch(`${url}/publishproject/${this.props.edit.project.id}`, {
+                method: "GET",
+                headers: {token: localStorage.token,
+                    "Content-Type": "application/json; charset=utf-8"}
+            })
+            .then(response => response.json())
+            .then(response => console.log(response))
```
[the remark this answers is not in the document]
Comment (marberrym) [reply src/pagescreens/EditProjectScreen.js]: fixed

File: routes/publishProject.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const db = require('../database');
+
+let publishProject = (req, res) => {
+    db.one(`UPDATE diy_projects SET publish_status=5 WHERE id=${req.params.id}`)
+    .then(res.send({status: 'successfully published'}))
```
[the remark this answers is not in the document]
Comment (marberrym) [reply routes/publishProject.js]: fixed

File: routes/publishProject.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+    .catch(err => console.log(err));
```
[the remark this answers is not in the document]
Comment (marberrym) [reply routes/publishProject.js]: fixed

File: src/pagescreens/EditProjectScreen.js [hunk 5]
```diff
@@ -88,6 +99,10 @@ class EditProjectScreen extends Component {
                             steptitle: '',
                             stepdescription: '',
                             stepimage: '',});
+            this.props.dispatch({type: "SET_TOAST", toast: {
+                type: 'info',
+                text: 'You added a new step!'
+            }})
         }
```
Comment (iangornall) [inline src/pagescreens/EditProjectScreen.js]: this toast may be annoying to user.

**Pull request merged**